src/angular/src/app/spark-core-angular/components/sprk-list-item/sprk-list-item.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SparkListItemComponent } from './sprk-list-item.component';
+
+describe('SparkListItemComponent', () => {
+  let component: SparkListItemComponent;
+  let fixture: ComponentFixture<SparkListItemComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ SparkListItemComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SparkListItemComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
should have a test for setting the analyticsString and for the additionalClasses logic